lib/tools/adb-emu-commands.js

```diff
@@ -31,6 +36,26 @@ emuMethods.rotate = async function () {
   await this.adbExecEmu(['rotate']);
 };
 
+emuMethods.powerAC = async function (state = 'on') {
+  if (_.values(emuMethods.POWER_AC_STATES).indexOf(state) === -1) {
+    log.errorAndThrow(`Wrong power AC state sent '${state}'. Supported values: ${_.values(emuMethods.POWER_AC_STATES)}]`);
+  }
+  await this.adbExecEmu(['power', 'ac', state]);
+};
+
+emuMethods.powerCapacity = async function (percent = 100) {
+  percent = parseInt(percent, 10);
+  if (_.isNaN(percent) || percent < 0 || percent > 100) {
```

Comment:
nit: It would be better here to use the standard JS's **isNaN** instead of the one from lodash lib.